test/AutoDiff/SILOptimizer/forward_mode_diagnostics.swift

```diff
@@ -85,10 +85,8 @@ func activeInoutParamControlFlow(_ array: [Float]) -> Float {
   return result
 }
 
-// FIXME(TF-984): Forward-mode crash due to unset tangent buffer.
```

Comment:
There are more `// FIXME(TF-984)` references in the autodiff test suite, could you please handle all of them? You can find them via `grep -nr "TF-984" test/AutoDiff`.